app/src/main/scala/org/alephium/explorer/persistence/schema/CustomGetResult.scala

```diff
@@ -80,6 +80,21 @@ object CustomGetResult {
       U256.unsafe(result.nextBigDecimal().toBigInt.bigInteger)
     }
 
+  @SuppressWarnings(Array("org.wartremover.warts.OptionPartial"))
+  implicit lazy val appStateValueGetResult: GetResult[AppState.Value] =
+    (result: PositionedResult) =>
+      AppState.Value.deserialize(ByteString.fromArrayUnsafe(result.nextBytes())).toOption.get
+
+  @SuppressWarnings(
+    Array("org.wartremover.warts.OptionPartial", "org.wartremover.warts.AsInstanceOf"))
+  implicit lazy val appStateTimeGetResult: GetResult[AppState.Time] =
+    (result: PositionedResult) =>
+      AppState.Value
+        .deserialize(ByteString.fromArrayUnsafe(result.nextBytes()))
+        .toOption
+        .get
+        .asInstanceOf[AppState.Time]
```

Comment:
Same as above for `toOption.get`. 

Following the minor change in the Serialiser code above to remove this typecast `.asInstanceOf[AppState.Time]`.

```scala
implicit val appStateTimeGetResult: GetResult[AppState.Time] =
  (result: PositionedResult) =>
    AppState.Time
      .deserialize(ByteString.fromArrayUnsafe(result.nextBytes()))
      .fold(throw _, value => value)
```